Utils/Elasticsearch/tools/daily/es_backup.logrotate

```diff
@@ -0,0 +1,46 @@
+nocreate
+nocompress
+nomail
+missingok
+dateext
+extension .tar.gz
+
+/var/es/backup/data.tar.gz {
+    daily
+    rotate 7
+    prerotate
+        base_dir=/var/es/backup
+        # create dirs for weekly/monthly/yearly copies
+        for d in weekly monthly yearly; do
+            mkdir -p -m 0700 "${base_dir}/$d"
+        done
+
+        # copy rotated file:
+        # - to weekly backups every Sunday
+         [ $(date "+%u") -eq 7 ] && ln "$1" "$base_dir"/weekly/"`basename $1`"
```

Comment:
It is a good idea to put _\`basename $1\`_ to own variable.

Reply:
Done: a8d5c65c03b3c65567a39920acdd6a5436136564